### DISCUSSION TITLE
Is there a pragma to waive checks for specific lines or modules in slang-tidy?

### DISCUSSION THREAD

**Discussion by TJackhammer** [MikePopoloski/slang#843] [Q&A]
I am using `slang-tidy` to do some style linting on my codebase. Generally, I want to flag all unintended implied latches, but there is one module that intentionally contains a latch -- is there a way to use a pragma to allow for `slang-tidy` to allow latches in a single module or something like that?

**Comment by MikePopoloski** [comment]
I do not think there is such a pragma currently, but it seems like a useful thing to add. I'm happy to accept a PR, or @Sustrak is the person who contributed / maintains the slang-tidy code currently.

**Comment by Sustrak** [comment]
Hi, sorry for the late replay. As Mike said, currently `slang-tidy` do not support pragmas. But you can disable all checks for a specific file. I take note of the pragma feature and I'll try to add it when I have some spare time. Glad to hear that people is using the linting tool altough it doesn't have so many checks for now 😄
